CMakeLists.txt

```diff
@@ -218,6 +218,10 @@ add_definitions( -D_WIN32_WINNT=0x0600)
 add_definitions( -DWINVER=0x0600)
 endif( WIN32 )
 
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Yay, \o/  finally i'd be able to use auto in lamda :-)

I guess we can do that because we stopped supporting these old debian and centOS that did not even have GCC 4.8 
although this is not strictly required for your patch.

Reply:
no - not required - but much more fun coding ;-)

Reply:
FYI @jnweiger @dschmidt compilers